### ISSUE TITLE
Enable functionality to choose name of created shortcut

### ISSUE THREAD

**Issue by sejonlu** [karakun/OpenWebStart#501]
There does not seem to be any feature that allows you to set the exact name of the created shortcut in a JNLP file. Could this be implemented?

Background: I am having an issue where if the user renames an old JNLP-generated desktop shortcut, OWS appears to interpret this as a desktop shortcut not existing and will therefore ask the user create a new desktop shortcut.

**Comment by sclassen** [comment]
Yes this is the behavior of OWS as it is infeasible to find all shortcuts if a user renames/moves them.
The name of the shortcut is determined by the `<title>` tag in the JNLP file.
If no title is present then the name of the file JNLP file is taken
